src/OmniSharp.Script/ScriptProjectSystem.cs

```diff
@@ -43,7 +41,24 @@ public class ScriptProjectSystem : IProjectSystem
             _env = env;
             _logger = loggerFactory.CreateLogger<ScriptProjectSystem>();
             _projects = new Dictionary<string, ProjectInfo>();
-            _scriptProjectProvider = ScriptProjectProvider.Create(loggerFactory);
+
+            _compilationDependencyResolver = new CompilationDependencyResolver(type =>
+            {
+                // Prefix with "OmniSharp" so that we make it through the log filter.
+                var categoryName = $"OmniSharp.Script.{type.FullName}";
+                var dependencyResolverLogger = loggerFactory.CreateLogger(categoryName);
+                return ((level, message) =>
+                {
+                    if (level == LogLevel.Debug)
+                    {
+                        dependencyResolverLogger.LogDebug(message);
+                    }
+                    if (level == LogLevel.Info)
```

Comment:
Are there any other related log levels (that matter?)

Reply:
No, that’s it. No other levels 😊

Reply:
Does this mean it will only log informational messages if the LogLevel == Info? I would expect it to send informational messages if LogLevel <= Info. Same for debug.

Reply:
There are only two possible levels logged from `Dotnet.Script.DependencyModel`

* Debug = 0
* Info = 1

> I would expect it to send informational messages if LogLevel <= Info.

`Debug` is the lowest level and it would sound a little strange to me to log that as a informational message? 

The `LogLevel` that we pass into the delegate is not from `Microsoft.Extensions.Logging`. It is defined in `Dotnet.Script.DependencyModel` with only those two levels. 

 `Dotnet.Script.DependencyModel` does not depend on `Microsoft.Extensions.Logging`. The code here simply forwards log messages to the actual logging framework which for OmniSharp.Roslyn means  `Microsoft.Extensions.Logging` 

Did that make any sense ?

Reply:
I see, I thought these were coming from `Microsoft.Extensions.Logging`.

Reply:
Figured you might think so:) Didn’t want to introduce a shared library version problem again. Therefore we don’t reference Microsoft.Extensions.Logging.